android/src/main/java/co/boundstate/BranchDeepLinks.java

```diff
@@ -106,7 +106,7 @@ private void sendError(String error) {
     public void handleUrl(PluginCall call) {
         // https://help.branch.io/developers-hub/docs/android-advanced-features#section-handle-links-in-your-own-app
 
-        String branchUrl = call.getString("url");
+        String branchUrl = call.getString("branch");
```

Comment:
I'm working on getting this test app setup. But first, I was wondering why this string was updated from "url" to "branch"